### ISSUE TITLE
Service is forbidden error in UI

### ISSUE THREAD

**Issue by SnuK87** [operator-framework/operator-lifecycle-manager#1245]
## Type of question
Asking for help

## Question
The UI shows an error `services is forbidden: User "system:serviceaccount:kube-system:default" cannot list resource "services" in API group "" in the namespace "default"`.

Is there any documentation about how to set / change / add a service account with proper privileges or what is the best way do solve this issue?

**What did you do?**
- Installed OLM on my k8s cluster using the scripted installation of release [0.13.0](https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/0.13.0).
- Executed `make run-console-local` to start the UI

**What did you expect to see?**
I expected the UI to work

**What did you see instead? Under which circumstances?**
An error message `services is forbidden: User "system:serviceaccount:kube-system:default" cannot list resource "services" in API group "" in the namespace "default"`

**Environment**
* operator-lifecycle-manager version:
https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/0.13.0

* Kubernetes version information:
```
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.0", GitCommit:"70132b0f130acc0bed193d9ba59dd186f0e634cf", GitTreeState:"clean", BuildDate:"2019-12-07T21:20:10Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.1", GitCommit:"d224476cd0730baca2b6e357d144171ed74192d6", GitTreeState:"clean", BuildDate:"2020-01-14T20:56:50Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}
```
  

* Kubernetes cluster kind: 
Bare metal

**Comment by chowarth123** [comment]
I have also tried to install the Operator Lifecycle Manager on baremetal kubernetes and I've got the same error when I run the UI and would also like help understanding how to fix the issue.

`Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.10", GitCommit:"7a578febe155a7366767abce40d8a16795a96371", GitTreeState:"clean", BuildDate:"2019-05-01T04:14:38Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.10", GitCommit:"7a578febe155a7366767abce40d8a16795a96371", GitTreeState:"clean", BuildDate:"2019-05-01T04:05:01Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}`

**Comment by chowarth123** [comment]
@SnuK87 not sure if you are still having this issue but I solved it on my environment. Here's my situation:

Baremetal kubernetes installation v1.11.10 and I've followed exactly the same steps that you did in your initial question and got the same error:
```
services is forbidden: User "system:serviceaccount:kube-system:default" cannot list resource "services" in API group "" in the namespace "default"
```

My understanding of the problem:
The "default" serviceaccount in the "kube-system" namespace doesn't have a role binding or a a cluster role binding and so it doesn't have the permissions it needs to query resources (services etc.). I'm guessing that Openshift gives the kube-system:default service account the permissions it needs and bare metal doesn't but I'm not an expert on Openshift...

My solution:
Create a `defaultrolebinding.yaml` file for a cluster role binding with the following content:
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: default-role-binding
subjects:
- kind: ServiceAccount
  name: default
  namespace: kube-system
roleRef:
  kind: ClusterRole
  name: cluster-admin
  apiGroup: rbac.authorization.k8s.io
```
Then kubectl apply it:
`kubectl apply -f defaultrolebinding.yaml`

The cluster-admin role probably has too much power but it'll give the OLM UI the power it needs at least.
